searx/external_bang.py

```diff
@@ -1,14 +1,73 @@
-from searx.data import bangs_loader
+# SPDX-License-Identifier: AGPL-3.0-or-later
 
-# bangs data coming from the following url convert to json with
-# https://raw.githubusercontent.com/jivesearch/jivesearch/master/bangs/bangs.toml
-# https://pseitz.github.io/toml-to-json-online-converter/
-# NOTE only use the get_bang_url
+from searx.data import EXTERNAL_BANGS
 
-bangs_data = {}
-for bang in bangs_loader()['bang']:
-    for trigger in bang["triggers"]:
-        bangs_data[trigger] = {x: y for x, y in bang.items() if x != "triggers"}
+
+def get_node(bang):
```

Comment:
Please pass `EXTERNAL_BANGS` as a parameter so these functions can be tested with a smaller list of test bangs.

Reply:
Done. 
This signature is `get_node(external_bangs_db, bang):`